cfgov/jinja2/v1/your-story/index.html

```diff
@@ -0,0 +1,302 @@
+                        <p>We want to hear from you about positive or negative experiences
+                           you’ve had with the financial products and services in your life.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="content-l_col content-l_col-1">
+                <div class="m-info-unit m-info-unit__inline">
+                    <div class="m-info-unit_image m-info-unit_image__square"
+                    style="background-image: url( {{ static('nemo/_/img/isocons-alt-green-review-and-route.png' ) }} );">
+                    </div>
+                    <div class="m-info-unit_content">
+                        <h3>2. We read your story</h3>
+                        <p>Someone at the CFPB will read your story. We take steps to protect
+                        your personal information, and your story is never shared with the
+                        public without your permission.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="content-l_col content-l_col-1">
+                <div class="m-info-unit m-info-unit__inline">
+                    <div class="m-info-unit_image m-info-unit_image__square"
+                    style="background-image: url( {{ static('nemo/_/img/isocons-alt-green-analyze-and-report.png' ) }} );"></div>
+                    <div class="m-info-unit_content">
+                        <h3>3. Your story helps others</h3>
+                        <p>The more we hear from you, the more insight we gain into what’s
+                        happening in the financial world and how it’s affecting you. We see
+                        trends and work to head off problems. <a title="Learn about enforcement actions" href="/policy-compliance/enforcement/actions/">Learn about our recent enforcement actions</a>.
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="block
+                block__flush-bottom
+                block__padded-bottom">
+        <div>Have a story to share with the CFPB?</div>
+        <a class="a-btn a-btn__full-on-xs u-mt15"
+           href="/your-story">
+            Tell your story
+        </a>
+    </div>
+</div>
+{% endblock %}
+
+{% block content_sidebar -%}
+<div class="consumer-stories">
+    <header class="m-slug-header">
+        <h2 class="a-heading">
+            EVERYONE HAS A STORY
+        </h2>
+    </header>
+    <ul class="m-list
+               m-list__unstyled
+               m-list__links">
+        <li class="m-list_item">
+            <p>
+                Learn how the CFPB has helped others take action
+                to address their financial problems while helping
+                them feel like they were not alone.
+            </p>
+        </li>
+        <li class="m-list_link">
```

Comment:
According to the CF docs this should be `m-list_item` and the `m-list_link` class should move to the link inside the li.